examples/ecs/capacity-provider-service-with-cluster-default-strategies/index.ts

```diff
@@ -4,15 +4,7 @@ import * as classic from "@pulumi/awsx/classic";
 
 const vpc = classic.ec2.Vpc.getDefault();
 
-const cluster = new classic.ecs.Cluster("cluster", {
-    capacityProviders: ["FARGATE_SPOT"],
-    defaultCapacityProviderStrategies: [
-        {
-            capacityProvider: "FARGATE_SPOT",
-            weight: 1,
-        },
-    ],
-});
+const cluster = new classic.ecs.Cluster("cluster");
```

Comment:
Curious about this change, does "cluster-default-strategies" still make a good example name after these are removed?